src/content/blog/astro-430.mdx

```diff
@@ -0,0 +1,127 @@
+publishDate: "February 2, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+coverImage: "/src/content/blog/_images/astro-430/post-header-4.3.webp"
+socialImage: "/src/content/blog/_images/astro-430/og-image-4.3.webp"
+lang: "en"
+---
+
+Astro 4.3 is now available! This release includes a new experimental i18n feature to try out, and improvements to working with your build output, component prop types, Markdown images, and more.
+
+Highlights include:
+
+- **[Add domain support for i18n](#experimental-add-domain-support-for-i18n)**
+- **[Add support for `build.format: 'preserve'`](#add-support-for-buildformat-preserve)**
+- **[Add `ComponentProps` type utility](#add-componentprops-type-utility)**
+- **[Better support for relative images in Markdown](#fix-using-images-in-markdown-without-a-relative-specifier)**
+
+## How to upgrade
+
+To take advantage of the latest features, make sure you're running the latest version of Astro. You can upgrade to Astro 4.3 by running the `@astrojs/upgrade` command:
+
+```sh
+npx @astrojs/upgrade
+```
+
+or by running the upgrade command for your package manager:
+
+```sh
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## (Experimental) Add domain support for i18n
+
+Astro 4.3 adds an experimental `domains` i18n configuration. This allows you to specify different domains or subdomains for different supported locales.
+
+For example, you can now use `example.com` for your English site, `fr.example.com` for your French site, and `example.es` for your Spanish site.
+Enable the experimental flag `i18nDomains` and map any or all of your locales to domains using `i18n.domains` in your `astro.config.mjs` file:
+
+```diff lang="js"
+// astro.config.mjs
+import {defineConfig} from "astro/config"
+
+export default defineConfig({
+    site: "https://example.com",
+    output: "server", // required, with no prerendered pages
+    adapter: node({
+      mode: 'standalone',
+    }),
+    i18n: {
+        defaultLocaLe: 'en',
+        locales: ['en', 'es', 'pt_BR', 'pt', 'fr'],
++        domains: {
++            fr: "https://fr.example.com",
++            es: "https://example.es"
++        },
+        routing: {
+          prefixDefaultLocale: true,
+        }
+    },
++    experimental: {
++        i18nDomains: true
++    },
+})
+```
+
+Note that this feature requires an entirely server-rendered site with no prerendered pages. Currently, the `@astrojs/node` and `@astrojs/vercel` adapters are supported with more adapter compatibility to come!
+
+See more in our [internationalization documentation](https://docs.astro.build/en/guides/internationalization/#domains-experimental) for more details and limitations on this experimental routing feature.
+
+## Add support for `build.format: 'preserve'`
```

Comment:
`build.format: 'preserve'` is kind of not a feature we support, but a config option that results in support for something. Suggesting this alternative (H/T @sarah11918)

```suggestion
## More control over your HTML file output
```